plugins/rollbar-backend/src/service/auto-assign/type.ts

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Config} from '@backstage/config';
+import {Logger} from 'winston';
+import {Router} from "express";
+import {RollbarApi} from "../../api";
+
+export interface AutoAssignOptions {
+  rollbarApi: RollbarApi;
+  logger: Logger;
+  rollbarCheck: RollbarCheckers
+  router: Router
+}
+
+
+// Rollbar API types. Just declared enough to use internally
+export type RollbarOccurrenceTraceFrame = {
```

Comment:
I choose create a specific type tree for my assign feature (because some types are different), WDYT?